components/raftstore-v2/src/operation/life.rs

```diff
@@ -89,6 +90,50 @@ impl DestroyProgress {
 }
 
 impl Store {
+    /// The method is called during split.
+    /// The creation process is:
+    /// 1. create an uninitialized peer if not existed before
+    /// 2. initialize the peer by the information sent from parent peer
+    #[inline]
+    pub fn on_split_init<EK, ER, T>(
+        &mut self,
+        ctx: &mut StoreContext<EK, ER, T>,
+        msg: Box<SplitInit>,
+    ) where
+        EK: KvEngine,
+        ER: RaftEngine,
+    {
+        let region_id = msg.region.id;
+        let mut raft_msg = Box::new(RaftMessage::default());
+        raft_msg.set_region_id(region_id);
+        raft_msg.set_region_epoch(msg.region.get_region_epoch().clone());
+        // Set the peer_id be INVALID_ID so that the message will not be sent to the
+        // peer after the creation.
+        raft_msg.mut_from_peer().set_id(raft::INVALID_ID);
+        raft_msg.set_to_peer(
+            msg.region
+                .get_peers()
+                .iter()
+                .find(|p| p.get_store_id() == self.store_id())
+                .unwrap()
+                .clone(),
+        );
+
+        // It will create the peer if it does not exist
+        self.on_raft_message(ctx, raft_msg);
+
+        ctx.router
+            .force_send(region_id, PeerMsg::SplitInit(msg))
+            .unwrap_or_else(|e| {
+                panic!(
```

Comment:
It's OK to return error. It's possible the peer is destroyed right before force send.